gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -37,8 +37,12 @@
               </p>
             </div>
             <div ng-if="editingExisting" class="input-group">
-              <div class="well well-sm text-monospace form-control" style="height: auto;" select-on-click>{{currentDevice.deviceID}}</div>
+              <div class="well well-sm text-monospace form-control">{{currentDevice.deviceID}}</div>
               <div class="input-group-btn">
+                <button type="button" class="btn btn-default" ng-click="copyToClipboard(currentDevice.deviceID)">
+                  <span class="fas fa-files-o"></span>&nbsp;<span class="ng-scope" translate>Copy</span>
+                </button>
+                <a class="btn btn-default" ng-click="copyToClipboard(currentDevice.deviceID)" href="mailto:?subject={{deviceName(currentDevice)}}%20has%20invited%20you%20to%20Syncthing&body={{deviceName(currentDevice)}}%20has%20invited%20you%20to%20Syncthing!%0D%0A%0D%0AYou%20can%20accept%20the%20invitation%20by%20adding%20a%20new%20device%20using%20the%20following%20ID.%20If%20unsure%20what%20to%20do,%20please%20continue%20reading%20below.%0D%0A%0D%0A{{currentDevice.deviceID}}%0D%0A%0D%0ASyncthing%20is%20an%20application%20that%20lets%20you%20synchronize%20your%20files%20across%20multiple%20devices.%20This%20means%20the%20creation,%20modification%20or%20deletion%20of%20files%20on%20one%20machine%20will%20automatically%20be%20replicated%20to%20your%20other%20devices.%20We%20believe%20your%20data%20is%20your%20data%20alone%20and%20you%20deserve%20to%20choose%20where%20it%20is%20stored.%20Therefore%20Syncthing%20does%20not%20upload%20your%20data%20to%20the%20cloud%20but%20exchanges%20your%20data%20across%20your%20machines%20as%20soon%20as%20they%20are%20online%20at%20the%20same%20time.%0D%0A%0D%0AFor%20more%20information%20and%20help%20on%20how%20to%20install%20and%20set%20up%20Syncthing,%20please%20visit%20our%20official%20website%20at%20https://syncthing.net.%0D%0A%0D%0AThe%20Syncthing%20Authors"><span class="fas fa-envelope-o"></span>&nbsp;<span class="ng-scope" translate>Email</span></a>
```

Comment:
`<button>` is impossible here, because `mailto` can be used inside `href` only. The copy to clipboard is a mistake (thanks for noticing!), but this is still a work in progress.

I'll see what I can then with moving the whole thing to JS.

Reply:
Moved everything to JS.